src/formatter/message-formatter.ts

```
@@ -13,82 +13,82 @@ import { formatData, styleData } from './data-formatter';
-            const dataString = formatData(data);
-            const styledData = styleData(dataString, colorScheme);
-            formattedMessage += styledData;
-        }
-        
-        // Always reset colors at the end of the entire log line
-        return formattedMessage + colors.reset;
+  static format(level: LogLevel, message: string, data?: any): string {
+    const { isoTimestamp, timeString } = getTimestampComponents();
+    const timestamp = formatTimestamp(isoTimestamp, timeString, colorScheme);
+
+    const levelName = this.getLevelName(level);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)